README.md

```diff
@@ -19,15 +19,19 @@ See more on our product webpage: <https://aspect.build/cli>
 
 # Installation
 
-## Homebrew (MacOS)
+## Bazelisk (MacOS / Linux)
```

Comment:
I think this repo should have both sections, or else some users will navigate here, install OSS, and wonder why starzelle isn't working.

Standard install
brew ...

OSS Code only
releases page ...

Reply:
Done